pkg/types/aliases.go

```diff
@@ -30,23 +30,31 @@ var Versions = []string{
 
 var LatestAddToScheme = directpv.AddToScheme
 
-type Interface = typeddirectpv.DirectpvV1beta1Interface
-type Client = typeddirectpv.DirectpvV1beta1Client
+type (
```

Comment:
This change has no effect on auto-generated code.

Reply:
we can gofumt this file after generation

Reply:
you need to `gofumpt`  `aliases.go.in`